pkg/util/mount/mount_windows.go

```diff
@@ -496,10 +496,11 @@ func getAllParentLinks(path string) ([]string, error) {
 
 // GetMountRefs : empty implementation here since there is no place to query all mount points on Windows
 func (mounter *Mounter) GetMountRefs(pathname string) ([]string, error) {
-	if _, err := os.Stat(normalizeWindowsPath(pathname)); os.IsNotExist(err) {
+	pathExists, pathErr := PathExists(normalizeWindowsPath(pathname))
+	if !pathExists {
```

Comment:
how about checking IsCorruptedMnt? This function should work for both windows and linux?

Reply:
https://github.com/kubernetes/kubernetes/pull/74625#discussion_r262214952

technically it "works" as in it compiles, but I think it is checking the incorrect error codes. Since there is no `XFS` on windows anyway we do not need to check for those types of corrupted mounts anyway